docs/explainers.md

```diff
@@ -0,0 +1,152 @@
+# Model Interpretability on Spark
+
+## Model Interpretability
+
+Model interpretability helps developers, data scientists and business stakeholders in the organization gain a comprehensive understanding of their machine learning models. It can also be used to debug models, explain predictions and enable auditing to meet compliance with regulatory requirements.
+
+## Why run model interpretation on Spark
+
+Model Interpretability on Spark enables users to interpret a black box model at massive scales with the Apache Spark™ distributed computing ecosystem. Various components support local interpretation for tabular, vector, image and text classification models, with two popular model-agnostic interpretation methods: Local Surrogate (LIME) and Kernel SHAP.
```

Comment:
Might want to mention that black-box interpretation can be computationally expensive due to the multiple evaluations needed to compute them.

Reply:
Done.